### PR TITLE
Feature/only office

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

File: docs/OnlyOffice.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
+.. _OnlyOffice:
+
+#############################
+OnlyOffice Connector for eXo
+#############################
+
+**What is OnlyOffice**
+
+`OnlyOffice Online Editors <https://www.onlyoffice.com/office-suite.aspx>`__ is a great open source solution 
+that lets you to create, edit  and collaborate online on your office documents.
```
Comment (thomasdelhomenie) [inline docs/OnlyOffice.rst]:
```suggestion
that lets you create, edit and collaborate online on your Office documents. 
```

File: docs/OnlyOffice.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+.. _EditorInterface:
+
+Editor Interface
+~~~~~~~~~~~~~~~~~~
+
+Clicking on |image5| button opens the document you want to edit in an another tab.
+The interface contains the content of the document loaded in the OnlyOffice editor with a floating button.
+
+Clicking on that button opens a drawer "Save version" with these fields:
```
Comment (thomasdelhomenie) [inline docs/OnlyOffice.rst:75]: Is "drawer" a understandable term for everyone ?

File: docs/OnlyOffice.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+-  The file's breadcrumb
+-  An edit icon allowing to edit the file's name
+-  The summary text field
+-  Save button allow to add a version of the document in Documents application.
```
Comment (thomasdelhomenie) [inline docs/OnlyOffice.rst]:
```suggestion
-  A ``Save`` button allowing to create a new version of the document .
```

File: docs/OnlyOffice.rst [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+-  List of the last previous 3 versions
```
Comment (thomasdelhomenie) [inline docs/OnlyOffice.rst]:
```suggestion
-  The list of the 3 latest versions
```

File: docs/OnlyOffice.rst [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+-  See more button
```
Comment (thomasdelhomenie) [inline docs/OnlyOffice.rst]:
```suggestion
-  A ``See more`` button allowing to display the full version history of the document
```

File: docs/OnlyOffice.rst [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+-  A close button at the top right of the drawer
+
+|image6|
+
+File's breadcrumb
+------------------
+
+The file's breadcrumb contains:
+
+-  The space's avatar if the file is on a space or the user's avatar if the file is on Personal documents
```
Comment (thomasdelhomenie) [inline docs/OnlyOffice.rst]:
```suggestion
-  The space's avatar if the file is in a space or the user's avatar if the file is in Personal documents 
```

File: docs/OnlyOffice.rst [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+-  The editor's avatar
+-  The changes timestamp, when hovering it a tooltip appears with the real edit date
+-  The edit summary which could be:
+
+   -  The summary inputted when saving the version
+   -  The text message "File has been updated" if the document's editor did not input a summary while saving the versions
+   -  The text message "File has been renamed to newname.extension" if the document's editor has renamed the file
+   
+Clicking on the ``See more`` button redirects you to the Documents application. YOu can then open Versions list by clicking
```
Comment (thomasdelhomenie) [inline docs/OnlyOffice.rst]:
```suggestion
Clicking on the ``See more`` button redirects you to the Documents application. You can then open Versions list by clicking 
```

File: docs/OnlyOffice.rst [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+-  The editor's avatar
+-  The changes timestamp, when hovering it a tooltip appears with the real edit date
+-  The edit summary which could be:
+
+   -  The summary inputted when saving the version
+   -  The text message "File has been updated" if the document's editor did not input a summary while saving the versions
+   -  The text message "File has been renamed to newname.extension" if the document's editor has renamed the file
+   
+Clicking on the ``See more`` button redirects you to the Documents application. YOu can then open Versions list by clicking 
+on Versions button on the toolbar.
```
Comment (thomasdelhomenie) [inline docs/OnlyOffice.rst]:
```suggestion
on ``Versions`` button on the toolbar.
```

File: docs/OnlyOffice.rst [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+
+|image9|
+
+Yellow bar
+~~~~~~~~~~~~
+
+When a document is opened in your stream, either in ``Documents`` application or in the activity stream, has been edited by
```
Comment (thomasdelhomenie) [inline docs/OnlyOffice.rst]:
```suggestion
When a document visible in your stream, either in ``Documents`` application or in the activity stream, has been edited by
```

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/OnlyOffice.rst (hunk 2 above)
Comment (mzorai) [reply docs/OnlyOffice.rst:75]: I think yes! any other proposals ?

Comment (thomasdelhomenie) [reply docs/OnlyOffice.rst:75]: If you think it is understandable, that's fine for me :) Otherwise, we could use "panel" maybe.

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**